.drone.yml

```diff
@@ -0,0 +1,43 @@
+---
+kind: pipeline
+name: amd64
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: validate
```

Comment:
I think this validate should check if `dist/` directory contains only semver compatible version, but the `scripts/validate` is just checking for `pkg/` directory.